example/lib/samples/beer_list_view.dart

```diff
@@ -48,6 +48,9 @@ class _BeerListViewState extends State<BeerListView> {
           pagingController: _pagingController,
           builderDelegate: PagedChildBuilderDelegate<BeerSummary>(
             animateTransitions: true,
+            firstPageErrorIndicatorBuilder: (_) => FirstPageErrorIndicator(
+              onTryAgain: () => _pagingController.refresh(),
+            ),
```

Comment:
is there a reason you have added this? 
this should be the default already, right?